### PR TITLE
WIP: feat(Upload): exposing isImageUrl as a parameter

### PR DESCRIPTION
**Pull request by muzea**
* [X] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
* [X] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [X] Rebase before creating a pull request to keep commit history clear.
* [X] Add some descriptions and refer relative issues for you pull request.

### CODE REVIEW

**Comment by zy410419243** [comment]
呃，你又手抖了...

**Comment by muzea** [comment]
:doge:
我还以为是让我重开分支的。。。。。

File: components/upload/interface.tsx [hunk 1]
```diff
@@ -93,4 +94,5 @@ export interface UploadListProps {
   showRemoveIcon?: boolean;
   showPreviewIcon?: boolean;
   locale: UploadLocale;
+  isImageUrl?: (file: UploadFile) => boolean;
```
Comment (zy410419243) [inline components/upload/interface.tsx]: 如果这个属性没用在 `state` 里，就不用加这行

**Comment by afc163** [comment]
feature branch

**Comment by afc163** [comment]
closing for no response

**Pull request closed without merging**